crates/rpc/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -152,4 +152,38 @@ impl Transaction {
             transaction_type: Some(U64::from(signed_tx.tx_type() as u8)),
         }
     }
+
+    /// Create a new rpc transaction result for a pending signed transaction, setting block
+    /// environment related fields to `None`.
+    pub fn into_transaction(self) -> TransactionSigned {
```

Comment:
Is this a mistaken addition (perhaps from a rebase)?

Reply:
Nvm, I see this is actually used, however I'm not sure I like it being here very much, esp since the next comment has some implicit behavior